tests/devices/test_default_mixed.py

```diff
@@ -26,11 +26,13 @@
     PauliX,
     Hadamard,
     CNOT,
+    SWAP,
     ISWAP,
     AmplitudeDamping,
     DepolarizingChannel,
     ResetError,
     PauliError,
+    MultiControlledX,
```

Comment:
Nice catch :+1:

Reply:
Yeah, this essentially is the test case that led to the reporting of #3582 .